src/tests/ftest/util/daos_server_utils.py

```diff
@@ -0,0 +1,80 @@
+"""
+  (C) Copyright 2022 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+from server_utils_base import DaosServerCommand
+
+
+class DaosServerCommandRunner(DaosServerCommand):
```

Comment:
I actually think it's better organization for `DaosServerCommandRunner` to be in the same file as `DaosServerCommand`. Though I'm not sure whether it makes more sense to move this new class to the existing file, or the existing class to this new file. I'm sure @phender has an opinion

Reply:
Done